frontend/packages/signless-transactions/src/components/create-session-modal/create-session-modal.tsx

```diff
@@ -87,39 +87,46 @@ function CreateSessionModal({ close }: Props) {
 
   return (
     <>
-      <Modal heading="Create Signless Session" close={close}>
+      <Modal heading="Enable Signless Session" close={close}>
         <ul className={styles.summary}>
-          <li>
+          <li className={styles['summary-item']}>
```

Comment:
```suggestion
          <li className={styles.summaryItem}>
```

let's stick to a single naming convention